pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -167,14 +168,17 @@ func NewNodePerformance(profile *performancev2.PerformanceProfile) (*tunedv1.Tun
 		}
 	}
 
-	if profile.Spec.WorkloadHints != nil {
-		if profile.Spec.WorkloadHints.RealTime != nil && *profile.Spec.WorkloadHints.RealTime {
-			templateArgs[templateRealTimeHint] = "true"
-		}
+	if IsRealTimeHintEnabled(profile) {
```

Comment:
minor, nonblocking: in general, refactoring changes should be in their own commit, and logic changes should be in their own separate commit.

Reply:
Yes sure, many changes before it, if necessary I can do it in a separate commit or PR but I did like that because is clearer even if mix differentes goals in the same commit.

Reply:
improving the current state (like extracting these helpers) and ensuring this change keeps the behavior consistent should be done separately from logic changes, because it makes review and git bisection easier. Anyway: not a big deal for this case, surely not worth a split *now*.

Reply:
thanks